tests/nodeos_contrl_c_test.py

```diff
@@ -98,16 +98,16 @@
     testSuccessful=False
 
     Print("Configure and launch txn generators")
-    nonProducerP2pPort = cluster.getNodeP2pPort(nonProdNode.nodeId)
     targetTpsPerGenerator = 500
```

Comment:
We can reduce this. Maybe 100 would be better.

Reply:
Happy to reduce it. Original aim was to maintain generation rates original attempted when using the trx_test_gen_plugin.

Reply:
Understood. Might as well reduce the load of this test while you are in here. The idea of the test is to have some transactions running during shutdown. 50 per block seems plenty for that requirement.